### PR TITLE
[css-shared-element-transitions-1] Add more non-normative text in the SET spec

### PR DESCRIPTION
**Pull request by vmpstr**
[css-shared-element-transitions-1] Add more non-normative text in the SET spec

### CODE REVIEW

File: css-shared-element-transitions-1/Overview.bs [hunk 1]
```diff
@@ -40,17 +40,92 @@ spec:css-shapes-3; type:function; text:rect()
...
+    the "after" state and position.
+* Wrapper pseudo-element: this element is a child of the container element and
+    provides ''isolation: isolate'' for its children. It's needed so that its
+    children can be blended with non-normal blend modes without affecting other
+    visual outputs.
+* Outgoing image: this element is a child of the wrapper element. It is a
+    replaced element that produced the visual representation of the "before"
+    state taken from user-agent provided snapshots. Note that the contents of
+    this element can be manipulated with ''object-*'' properties in the same way
+    that regular images can be.
```
Comment (khushalsagar) [inline css-shared-element-transitions-1/Overview.bs:99]:
```suggestion
    that other replaced elements can be.
```

File: css-shared-element-transitions-1/Overview.bs [hunk 2]
```diff
@@ -40,17 +40,92 @@ spec:css-shapes-3; type:function; text:rect()
...
+* Incoming image: this element is a child of the wrapper element. It is a
+    replaced element that produces the visual representation of the "after"
+    state, taken from the page's visual output of the represented elements. Like
+    outgoing image, the contents can be manipulated with ''object-*'' properties.
+    Note that because this element's snapshots are taken from the pages output,
+    the visual output changes if the visual output of the represented element
+    changes. This is similar to how an 'element()' function would work.
```
Comment (khushalsagar) [inline css-shared-element-transitions-1/Overview.bs:106]: Can we link to the element() spec here?

File: css-shared-element-transitions-1/Overview.bs [hunk 3]
```diff
@@ -40,17 +40,92 @@ spec:css-shapes-3; type:function; text:rect()
...
+
+ISSUE: This sections can benefit from diagrams.
+
+Each of the pseudo-elements generated can be targeted by CSS in order to
+customize its appearance and behavior. This enables full customization of the
```
Comment (khushalsagar) [inline css-shared-element-transitions-1/Overview.bs:111]:
```suggestion
customize its appearance, behavior and/or add animations. This enables full customization of the
```

File: css-shared-element-transitions-1/Overview.bs [hunk 4]
```diff
@@ -92,6 +167,17 @@ the following style in the [=user-agent origin=].
     }
  </code></pre>
 
+<div class=note>This property causes the user-agent to both capture separate
+snapshots from the elements, as well as create separate pseudo-element
+structure representing this element's "before" and "after" states. Note that
```
Comment (khushalsagar) [inline css-shared-element-transitions-1/Overview.bs:172]:
```suggestion
sub-trees representing this element's "before" and "after" states. Note that
```

**Review by khushalsagar — APPROVED** [review]
This looks great! Thanks for writing it.

File: css-shared-element-transitions-1/Overview.bs (hunk 2 above)
Comment (vmpstr) [reply css-shared-element-transitions-1/Overview.bs]: Done.

**Pull request merged**